tutorials/tutorial-6-k.py

```diff
@@ -0,0 +1,23 @@
+import pygem as pg
+from pygem.khandler import KHandler
+import vedo
+import meshio
+
+ffd = pg.FFD()
+ffd.read_parameters('../tests/test_datasets/parameters_test_ffd_pipe_unv_C0.prm')
+
+%cat ../tests/test_datasets/parameters_test_ffd_pipe_unv_C0.prm
```

Comment:
are you sure this line works in a python script?

Reply:
Works from Sypder, but not from the command line.  Therefore now gone! Thanks